app/actors/hyrax/actors/create_with_remote_files_actor.rb

```diff
@@ -0,0 +1,104 @@
+module Hyrax
+  module Actors
+    # Attaches remote files to the work
+    class CreateWithRemoteFilesActor < Hyrax::Actors::AbstractActor
```

Comment:
What change was needed here that required us overriding the (presumably) upstream `CreateWithRemoteFilesActor`?